includes/wf_crm_webform_postprocess.inc

```diff
@@ -1896,9 +1895,17 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
     // tax integration
     // TODO: needs review if this is changed in 4.6
     if (!is_null($this->tax_rate)) {
```

Comment:
We pulled in the non_deductible_amount edits in another PR a few days ago - so your Edits below are no longer necessary.